pages/docs/running/testing.mdx

```diff
@@ -101,3 +101,53 @@ describe("snapshot", () => {
 ## Testing Notes
 
 - Unless you specify in your `package.json` or `jest.config.js`, You'll need to use magic comments to let Jest know that we are running a Node.js app (`@jest-environment node`)
+
+## Spec Helper API
+
+The spec helper makes a few helpful methods available to you. Here are the commonly-used methods and options:
+
+### `grouparooTestServer`
+
+Starts a test version of the Grouparoo server prior to running the specs and stops it after running the tests.
+
+Usage:
+
+```js
+const options = {
+  truncate: false,
+  enableTestPlugin: false,
+  resetSettings: false,
+  disableTestPluginImport: false,
```

Comment:
I don't think that our customers will be using these options.  Maybe just don't list them?